.github/workflows/main.yml

```diff
@@ -105,9 +105,11 @@ jobs:
           $(go env GOPATH)/bin/ziti-ci configure-git
           $(go env GOPATH)/bin/ziti-ci generate-build-info common/version/info_generated.go version
           go install github.com/mitchellh/gox@latest
```

Comment:
Can we leave this section as is? I'm not a fan of the line splitting

Reply:
OK, I just sent a commit to the source branch to stop aligning the `gox` commands horizontally with line splitting.